### PR TITLE
adds page to view errors

### PR DESCRIPTION
**Pull request by hailed748**
Adds a page that lists out all the bugs that have been reported.

image

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/Report.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Entity newReport = new Entity("Report");
     newReport.setProperty("type", request.getParameter("report-type"));
     newReport.setProperty("report", request.getParameter("report"));
-    newReport.setProperty("user-email", request.getParameter("email"));
+    newReport.setProperty("user", request.getParameter("email"));
```
Comment (ninaprabhu) [inline src/main/java/com/google/sps/servlets/Report.java:22]: why did you change the name? I feel like user-email or email is more intuitive

Comment (hailed748) [reply src/main/java/com/google/sps/servlets/Report.java:22]: the dash became and issue when trying to pull the attribute because I am sending the entities over rather than making a whole new object. I can just change it to email

**Comment by LukeGMTravis** [comment]
Starting review
